host/linux/aesm.c

```diff
@@ -313,10 +313,11 @@ static oe_result_t _write_request(
     oe_result_t result = OE_UNEXPECTED;
     mem_t envelope = MEM_DYNAMIC_INIT;
 
-#if (OE_TRACE_LEVEL >= OE_TRACE_LEVEL_INFO)
-    OE_TRACE_INFO("=== _write_request:\n");
-    oe_hex_dump(mem_ptr(message), mem_size(message));
-#endif
+    if (get_current_logging_level() >= OE_LOG_LEVEL_INFO)
+    {
+        OE_TRACE_INFO("=== _write_request:\n");
```

Comment:
It seems like it OE_TRACE_INFO should check the current logging level internally.

Reply:
This was a straight macro replacement on the old code.
I moved OE_TRACE_INFO() call to right above line 316 to address your comment.